.github/workflows/alpine-32bit-build-and-test.yaml

```diff
@@ -0,0 +1,73 @@
+name: Linux 32-bit x86
+on:
+  push:
+#    branches:
+#      - prerelease_test
```

Comment:
Is there any specific reason to keep this commented? Is it correct that this workflow runs on any commit to any branch in the repo? Will it be better to do in the same way as @erimatnor did for Windows build, when workflows run on pull requests and push to prerelease branch?